src/index.js

```diff
@@ -90,6 +90,27 @@ const blockToText = (item, childrenRenderer, isTopLevelNode) => {
     return renderer(item, childrenRenderer, isTopLevelNode);
 };
 
+const blockToCsv = (item, childrenRenderer, isTopLevelNode, templateMetadata) => {
+    let renderer = null;
+    switch (item.type) {
+    case EDIE_BLOCK_TYPE.MAIN:
+        renderer = mainToCsv;
+        break;
+    case EDIE_BLOCK_TYPE.ROW:
+        renderer = rowToCsv;
+        break;
+    case EDIE_BLOCK_TYPE.LOOP:
+        renderer = loopToCsv;
+        break;
+    case EDIE_BLOCK_TYPE.COLUMN:
+        renderer = columnToCsv;
+        break;
+    default:
+        return `Conversion of ${item.type} to TEXT not implemented.`;
```

Comment:
"to TEXT" -> "to CSV"

Reply:
Oops. Good catch! Will change it. Thanks.

Reply:
Changed.